packages/node/src/eventbuilder.ts

```diff
@@ -105,7 +105,14 @@ export function eventFromMessage(
   if (attachStacktrace && hint && hint.syntheticException) {
     const frames = parseStackFrames(hint.syntheticException);
     if (frames.length) {
-      event.stacktrace = { frames };
+      event.exception = {
+        values: [
+          {
+            value: message,
+            stacktrace: { frames },
+          },
```

Comment:
As per https://develop.sentry.dev/sdk/event-payloads/types/#valueclass

>  At least one of type or value is required, otherwise the exception is discarded.

Since we have value, we are fine with not including type? We can update the event type accordingly if TS is complaining.

Reply:
ts doesn't care, I was just reading the dev docs and completely missed that part!